### PR TITLE
Add Planned Enhancements

### PR DESCRIPTION
**Pull request by rahula1008**
1. Add the planned enhancements
2. Add the description of the rare bug with SortedList that causes reordering of people with the same name upon changing of different field.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -460,6 +450,70 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Versioned AddressBook**: An extension to the AddressBook class that allows undoing/redoing actions.
 
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+**Note:** Our team size is 5
+
+1. **Make GUI wrap large addresses to the next line:** With the current implementation, if you type in an address which is longer than the width of the screen, it stretches the person card (for all contacts), forcing the user to scroll to the right to see the rest of the address, the project status and deadline. For future implementation, we plan to wrap the address, such that if it is very long, it wraps to the next line.
```
Comment (tanwartushar) [inline docs/DeveloperGuide.md:458]:
```suggestion
1. **Make the GUI wrap large addresses to the next line:** With the current implementation, if you type in an address which is longer than the width of the screen, it stretches the person card (for all contacts), forcing the user to scroll to the right to see the rest of the address, the project status and deadline. In future updates, we plan to enable `textWrap` and set a `maxWidth` for the address label to enhance the UI and improve readability.
```

**Review by Dinoman44 — COMMENTED** [review]
Few edits to make, but otherwise LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -460,6 +450,70 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. Observe the initial order
+3. Do a sort command (`sort name ascending`)
+4. Start editing other fields of these John's (i.e edit their project status)
+5. After doing some of these edits, you will most likely see the list order change
+
+In the future, we will firstly modify the program, such that it no longer uses SortedList.
+
+1. In ModelManager, delete the sortedPersons variable
+2. This will include modifying the getFilteredPersonList() method, to return filteredPersons, instead of sortedPersons.
+3. Replace the sortByComparator method to use the FXCollections.sort() method. This is sample code you can use:
```
[the remark this answers is not in the document]
Comment (rahula1008) [reply docs/DeveloperGuide.md:490]: Good idea!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -460,6 +450,70 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+8. **Make it such that modifications while viewing a filtered list (ie. After using a `find`/`blacklist`/`whitelist` command) will operate on the filtered list** Currently, after filtering a list, if you perform a command, (eg.`add`/`edit`/`delete` etc), these would all be operating on the main client list, NOT the filtered list. As such the UI will also reset to the main client list as well. In the future, we plan to make it such that while viewing a filtered list, commands will be operating on that particular filtered list.
+
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Sorted List Bug**
+
+Reported [here](https://bugs.openjdk.org/browse/JDK-8301761)
+
+Description of SortedList bug:
```
[the remark this answers is not in the document]
Comment (Dinoman44) [reply docs/DeveloperGuide.md:474]: I'm dumb my bad

**Review by Dinoman44 — APPROVED** [review]

**Pull request merged**